scale-testing/scale.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+import subprocess, os
+from subprocess import Popen, PIPE
+import time
+import json
+import test_request
+
+
+"""
+THIS IS A DESCRIPTION OF WHAT THIS CLASS DOES
+"""
+
+
+class scale:
+
+    runtime = 0
+
+    cpu_util = 0
+
+    mem_util = 0
+
+    throughput = 0
+
+    CMD = ""
+    
+    def __init__(self, pname, pid):
+        """
+        Initialize 
+        """
+
+        self.name = pname
+        self.pid = pid
+        self.CMD = ""
+        
+
+    def set_init(self, pname, pid, cmd):
+        """
+        Set name and process ID
+        """
+
+        self.name = pname
+        self.pid = pid
+        self.CMD = cmd
+
+
+    def set_cmd(self, cmd):
+        """
+        Set CMD only; e.g. use same script (& process & processID) but on different parameters
+        """
+
+        self.CMD = cmd
+        
+        
+    def get_uinput(self):
+        """
+        Query user for information about the command to be run
+        """
+
+        self.name = raw_input("Enter the process name: ")
+        self.pid = raw_input("Enter the process ID: ")
+
+#        print "Please ensure that all necessary dependencies and configurations are adequately prepared and all necessary files are present in this directory"
+        self.CMD = raw_input("Enter the command you intend to run: ")
+        
+        
+    def run(self):
+        """
+        Run the desired command and measure response time, throughput, and CPU/memory utilization
+        """
+
+        print "\nStarting to execute command"
+        print "____________________________\n\n"
+
+        # clear any preexisting logs                                                                                                                                                       
+        subprocess.call('rm /tmp/top.log', shell=True)
+
+        birth = time.time()
+
+        subprocess.call(["top -p " + str(self.pid) + " -n 1 | grep " + self.name + " | awk '{print $9, $10}' >> /tmp/top.log"], shell=True)
```

Comment:
Again with too many line breaks.  Don't use them unless you are separating meaningful sections.